twilight-lavalink/Cargo.toml

```
@@ -42,6 +45,7 @@ http-support = ["dep:percent-encoding"]
 native-tls = ["tokio-websockets/native-tls", "tokio-websockets/openssl"]
 rustls-native-roots = ["tokio-websockets/ring", "tokio-websockets/rustls-native-roots"]
 rustls-webpki-roots = ["tokio-websockets/ring", "tokio-websockets/rustls-webpki-roots"]
+lavalink-protocol-http2 = []
```

Conditionally enable hyper's features based on this.
```suggestion
http2 = ["hyper/http2", "hyper-util/http2"]
```